hack/e2e.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+#
+# This script assumes its running from a pod and does the following:
+# * Processes the template to setup a local cincy instance
+# * Prepares a graph using test data
+# * Ensures generated graph is valid
+
+# Debug information
+oc whoami
+oc project
+
+# Apply oc template
+oc process -f ../dist/openshift/cincinnati.yaml \
+  -p IMAGE=${CINCINNATI_E2E_IMAGE:-pipeline} \
+  -p IMAGE_TAG=${CINCINNATI_E2E_IMAGE_TAG:-deploy} \
+  -p GB_CINCINNATI_REPO="redhat/openshift-cincinnati-test-public-manual"
```

Comment:
AFAIU you are trying to run the Cincy instance here, right? then `oc process` does not create an application from a template.

Reply:
https://docs.openshift.com/container-platform/3.11/dev_guide/application_lifecycle/new_app.html#specifying-a-template

Reply:
ah, good catch. Fixed by adding `| oc create -f`

Reply:
From those docs:

> To directly use a template in your local file system, without first storing it in OpenShift Container Platform, use the `-f|--file` argument...

which is what we have, right?